### PR TITLE
Add role of group member

### PR DESCRIPTION
**Pull request by rdvhoorn**

### CODE REVIEW

File: src/js/user/edit.js [hunk 1]
```diff
@@ -70,10 +91,11 @@ app.controller("userEditController", ["$scope", "$q", "$routeParams", "users", "
             $scope.user.isAdmin = $scope.selectedRole.id === "Admin";
             $scope.user.displayName = $scope.user.firstName + " " + $scope.user.lastName;
             $scope.loading = true;
+            console.log($scope);
             user.edit($scope.user, $scope.groupSelection).then(function (result) {
                 $scope.loading = false;
                 // redirect to '/manage'
-                window.location.href = "/manage";
+                // window.location.href = "/manage";
```
Comment (HiddeKoerts) [inline src/js/user/edit.js]: why disabled?

File: src/js/user/edit.js [hunk 2]
```diff
@@ -70,10 +91,11 @@ app.controller("userEditController", ["$scope", "$q", "$routeParams", "users", "
             $scope.user.isAdmin = $scope.selectedRole.id === "Admin";
             $scope.user.displayName = $scope.user.firstName + " " + $scope.user.lastName;
             $scope.loading = true;
+            console.log($scope);
```
Comment (HiddeKoerts) [inline src/js/user/edit.js]: definitely shouldn't be there

File: src/www/templates/group/groupDetails.html [hunk 3]
```diff
@@ -6,5 +6,5 @@ <h4>Group description</h4>
     <h4>Contact details</h4>
     <p>{{group.email}}</p>
     <h4>Members</h4>
-    <p ng-repeat="member in group.members">{{member.displayName}}</p>
+    <p ng-repeat="member in group.members">{{member.displayName}} ({{member.user_group.func}})</p>
```
Comment (HiddeKoerts) [inline src/www/templates/group/groupDetails.html:9]: maybe an idea to sort on function?

File: src/js/user/edit.js [hunk 4]
```diff
@@ -44,13 +46,32 @@ app.controller("userEditController", ["$scope", "$q", "$routeParams", "users", "
...
+            }
+            assert(false) // this can't/shouldn't happen happen
+        };
+
         // get attributes of groups of which user is member
         $scope.groupSelection = [];
         for (i = 0; i < $scope.groups.length; i++) {
             if (member_groups.includes($scope.groups[i].id)) {
-                $scope.groupSelection.push({fullName: $scope.groups[i].fullName, id: $scope.groups[i].id, selected: true})
+                $scope.groupSelection.push({
```
Comment (HiddeKoerts) [inline src/js/user/edit.js:67]: more in general: comments are nice :)

File: src/www/templates/group/groupDetails.html (hunk 3 above)
Comment (rdvhoorn) [reply src/www/templates/group/groupDetails.html:9]: Did so but is not super clean

File: src/js/user/edit.js (hunk 4 above)
Comment (rdvhoorn) [reply src/js/user/edit.js:67]: Added some comments :)

**Review by HiddeKoerts — APPROVED** [review]

**Pull request merged**